src/raylib.h

```diff
@@ -297,11 +297,13 @@ typedef struct Font {
 
 // Camera, defines position/orientation in 3d space
 typedef struct Camera3D {
-    Vector3 position;       // Camera position
-    Vector3 target;         // Camera target it looks-at
-    Vector3 up;             // Camera up vector (rotation over its axis)
-    float fovy;             // Camera field-of-view apperture in Y (degrees) in perspective, used as near plane width in orthographic
-    int projection;         // Camera projection: CAMERA_PERSPECTIVE or CAMERA_ORTHOGRAPHIC
+    int mode;
```

Comment:
I'm afraid I have no plans to change current camera structure. Provided parameters are clear and standard for users. It will also break many codebases.

Reply:
@Crydsch I'm afraid changing the current `Camera` structure is a breaking change and it could break many codebases and implementations. I also find the new implementation more confusing for users than current one, raylib prioritize simplicity for less-experienced users and current Camera provides that. I think a `Quaternion orientation` approach is confusing for many developers starting with raylib, probably not for more advance users.

In any case, I think current camera parameters could be kept as they are and transformations could be done inside the update code.

Reply:
Well a camera example like the other one already available could just fix the problem, i agree that the current implementation is just really simple, i would prefer a more professional way than a limited camera, its only my answer through, you can create as many example to ease the breaking change.